lib/core/src/display/scene/htmlscene.rs

```diff
@@ -0,0 +1,50 @@
+use super::HTMLObject;
```

Comment:
Format this file according to previous comments, make lines max 80 chars, make more vars, so you dont have multi-line exprs after refactoring, and this will be a very nice file! <3

Reply:
A newer file formatted with `cargo fmt` should be available. Would you check it?